cmd/repl/prompt.go

```diff
@@ -84,7 +84,7 @@ func (p *promptState) changeLivePrefix() (string, bool) {
 
 func (p *promptState) execute(in string, cb func(string)) {
 	in = strings.TrimRight(in, " \t")
-	if in != "" {
+	if in != "" && !(p.statement == "" && strings.HasPrefix(strings.TrimSpace(in), "-- ")) {
```

Comment:
I think we need unit test for function execute.

Reply:
Done.